index.py

```diff
@@ -529,10 +530,7 @@ def sendMail(smtp, parts, from_addr, from_name, to_addr, subject, msgid=None, in
             if part['subtype'] == 'plain':
                 alt.set_param('format', 'flowed')
             msg.attach(alt)
-    readable_from = email.header.Header(charset='utf8', header_name='From')
-    readable_from.append(from_name)
-    readable_from.append('<%s>' % (from_addr), charset='us-ascii')
-    msg['From'] = readable_from
+    msg['From'] = formataddr((str(Header(from_name,'utf-8')),from_addr))
```

Comment:
There seems to a superfluous level of parenthesis here.

The above line should be:
```python
msg['From'] = formataddr(str(Header(from_name,'utf-8')),from_addr)
```

Reply:
no [`formataddr()̀`](https://docs.python.org/3/library/email.utils.html) indeed takes a pair as first argument